src/libs/UnreadIndicatorUpdater/index.js

```diff
@@ -11,7 +12,14 @@ const unreadActionCounts = {};
  * and Mac OS or iOS dock icon with an unread indicator.
  */
 const throttledUpdatePageTitleAndUnreadCount = _.throttle(() => {
-    const totalCount = _.reduce(unreadActionCounts, (total, reportCount) => total + reportCount, 0);
+    // If all of our nonzero unread action counts show -1, update the unread count to be -1 as well so we don't show a
+    // number in the indicator badge.
```

Comment:
out of curiosity, how do we get a unread action count to be -1?

Reply:
Further down below in `listenForReportChanges` here:
https://github.com/Expensify/Expensify.cash/pull/3766/files#diff-0812bc9a1458d29651dc472eddb66ec3c320b32473461e75d8dda4fbd6362523R47